src/test/java/TestSuites.java

```diff
@@ -0,0 +1,120 @@
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class TestSuites {
+
+    private final Rectangle farmSquare = new Rectangle(40, 30, 130, 90);
```

Comment:
it would be great to have constants for most usable int literals. like FARM_LEFT_CORNER_X = 40 and so on